examples/querydbtable_config.yml

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Flow Controller:
```

Comment:
I am not even sure what it does but in the previous examples the files started with
` MiNiFi Config Version: 3`
We could be consistent with that and include it everywhere, or nowhere?

Reply:
It seems it is not used currently as we are on version 0.10.0. I suppose it will be only checked after version 1.0 to handle config versions for backward compatibility. I added the config version to all yamls in f45f0ed787ea86742bbd093a97d11006201f26c6